qa/os/windows-2012r2/build.gradle

```diff
@@ -8,6 +8,6 @@ if (boxId != null) {
 } else {
   // box id was not supplied, so disable the distro tests
   tasks.withType(GradleDistroTestTask).configureEach {
-    onlyIf { false }
+    onlyIf("always false") { false }
```

Comment:
I think this should actually say something like "Project property 'vagrant.windows-2012r2.id' set".

Reply:
fixed